test/vault.spec.ts

```diff
@@ -113,6 +108,26 @@ contract("Vault", (accounts) => {
       );
   };
 
+  async function deployTokenWithFee(fee: BigNumber, initialAccount: Address, from: Address = ownerAccount) {
```

Comment:
I wonder if there's a way to make this a library so there's no duplication across `vault.spec` and `transferProxy.spec`

Reply:
For sure, we will DRY up testing shortly.